### ISSUE TITLE
Feature request: provide option to disable link following

### ISSUE THREAD

**Issue by kenbarr1** [jrblevin/markdown-mode#290]
It markdown-mode, hyperlinks are clickable/followable -- which can make it frustrating/difficult to edit files because the buffer changes whenever I click in the wrong place.  

I have some workarounds, but it would be nice to expose this as an option that one could toggle.

Workaround:

```elisp
; Turn off click-follows-link in Markdown mode.
(defun disable-goto-addr ()
  (setq-local mouse-1-click-follows-link nil)
  ; turn middle clicks into the default action, normally pasting the primary selection.
  (define-key markdown-mode-mouse-map [mouse-2] nil)
)
(add-hook 'markdown-mode-hook 'disable-goto-addr)
```

**Comment by jrblevin** [comment]
Thanks, I will make this a user-customizable option in the future.

**Comment by syohex** [comment]
I have sent PR #500. How about it ?

**Comment by syohex** [comment]
I have merged #500. Please check latest version.
